onboarding.md

```diff
@@ -0,0 +1,43 @@
+# Onboarding
+
+This document covers steps needed to onboard new Steering Committee
+members and off-board emeritus members.
+
+- [ ] All new members must complete the [Inclusive Leadership Training].
+
+- [ ] Update the following files in the respective repos:
+  - [ ] [kubernetes/steering]: `OWNERS_ALIASES` and `README.md`
+  - [ ] [kubernetes/community]: `sigs.yaml` and auto-generated content
+  - [ ] [kubernetes/org]: `OWNERS_ALIASES` and the `steering-committee`
+    GitHub team
+  - [ ] [kubernetes/funding]: `OWNERS_ALIASES`
+  - [ ] [kubernetes/kubernetes-template-project]: `OWNERS_ALIASES`
+
+- [ ] Add new members with Owner access to the `steering-private`
+  mailing list and remove emeritus members.
+
+- [ ] Grant new members Owner access to the public steering mailing
+  list and revoke access from emeritus members.
+
+- [ ] Add new members to the `steering-private` slack channel and
+  remove emeritus members.
+
+- [ ] Add new members to the CNCF Service Desk and remove emeritus
+  members.
+
+- [ ] Add new members to the [cncf-kubernetes-maintainers] mailing list.
+
+- [ ] If a GSuite top-level account is owned by an emeritus member,
+  transfer it to an existing member.
+
+- [ ] If Zoom license credentials were owned by a majority of emeritus
```

Comment:
@parispittman Is `Zoom license credentials` the right term for the zoom creds? :upside_down_face: 

Also, how are the meetings uploaded to youtube today? Is it handled by SC members (akin to SIGs)? Do YouTube creds also need to be shared?

Reply:
thats right 

we have it automated right now with splain. as long as we hit 'record to the cloud', splain will pick it up and youtube admins will clean it up and post for us within 48 hours.